actions/changelog-generate/action.yaml

```diff
@@ -73,14 +74,15 @@ runs:
           fi
         fi
         echo  "fromtag=$setfrom">> $GITHUB_OUTPUT
+        echo "path=${{ inputs.config }}" >> "$GITHUB_OUTPUT"
```

Comment:
Why is the remapping to an output here needed instead of just `configuration: "${{ inputs.config }}"` below?

Reply:
As discussed, it is fine like this, because without the remapping it does not seem to work. Please leave a short comment above it, so we are not confused in the future